### ISSUE TITLE
font size slider

### ISSUE THREAD

**Issue by alltheseas** [damus-io/notedeck#514]
Some users may want to e.g. increase font size, as to be able to more easily see text

see iOS Damus as example.

**Comment by jb55** [comment]
we have ui zoom ?

**Comment by alltheseas** [comment]
> Was anyone successful making the text smaller? I know on desktop it’s super easy, but I’m not sure how to do it on phone.

> I noticed it varies wildly between devices. I wonder if we should be calculating dpi and sizing text this way somehow

> Just need a slider in settings like on ios
>
> Once we have a settings view…
